lang/java/reef-runtime-local/src/main/java/org/apache/reef/runtime/local/client/parameters/RootFolder.java

```diff
@@ -24,7 +24,7 @@
 /**
  * The folder where logs etc. shall be stored.
  */
-@NamedParameter(default_value = RootFolder.DEFAULT_VALUE, doc = "The folder where logs etc. shall be stored.")
+@NamedParameter(default_value = RootFolder.DEFAULT_VALUE, doc = "The folder where the logs and results are stored.")
 public final class RootFolder implements Name<String> {
   public static final String DEFAULT_VALUE = "REEF_LOCAL_RUNTIME";
```

Comment:
Running `HelloREEFStandalone` creates a folder named `REEF_LOCAL_RUNTIME`. Could you make the default name `REEF_STANDALONE_RUNTIME`, instead? I guess you would need to create a separate parameter for the standalone runtime, rather than using the local runtime's.